dist/static.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HackUPC | October 7th - 9th | Barcelona | 500 Hackers</title>
+  <meta property="og:title" content="HackUPC | October 7th - 9th | Barcelona | 500 Hackers"/>
+  <meta property="og:site_name" content="HackUPC"/>
+  <meta name="description" content="Join us for BarcelonaTech's hackathon. Hack in Barcelona with 500 hackers. 36h. October 7th-9th.">
+  <meta property="og:description" content="Join us for BarcelonaTech's hackathon. Hack in Barcelona with 500 hackers. 36h. October 7th-9th."/>
+  <meta property="og:image" content="https://hackupc.com/assets/img/hackupc-ogimage@2x.png"/>
+  <meta property="og:image:secure_url" content="https://hackupc.com/assets/img/hackupc-ogimage@2x.png"/>
+  <meta name="author" content="Hackers@UPC">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+  <link rel="stylesheet" href="assets/css/static.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Ek+Mukta" rel="stylesheet">
+</head>
+
+<body>
+    <div id="app">
+      <h1 class="col-sm-4 col-sm-offset-4" id="title">{{title}}</h1>
```

Comment:
It's too low and too small. Needs to be centered as HackUPC banner in index.html